### PR TITLE
Add import dapp/project functionality

### PR DESCRIPTION
**Pull request by Bashlund**
Added functionality to import an exported DApp.

_Test:_
1. Go to settings of a project, click `Show DAppfile.json for export`, copy that JSON.
2. Click `New DApp` button.
3. Enter a new project name.
4. Click the link at the bottom right on the modal.
5. Paste the JSON.
6. Click the `Import` button.

_Expected_:
New DApp should have been created as a clone of the first one.
Any state data will *not* have been included, such as compiled bytecode, deployed addresses, etc.
These states can however be included if we want that behavior.

### CODE REVIEW

File: src/components/newdapp/index.js [hunk 1]
```diff
@@ -93,6 +101,59 @@ export default class DevkitNewDapp extends Component {
...
+            if(!contentJSON) {
+                alert("No JSON provided. Paste it into the text area and try again.");
+                return;
+            }
+            var dappfileJSONObj;
+            try {
+                dappfileJSONObj=JSON.parse(contentJSON);
+            }
+            catch(e) {
+                alert("Could not parse JSON, is it properly pasted?");
```
Comment (filippsen) [inline src/components/newdapp/index.js]: Properly pasting is not a problem, per se. The problem is simply an invalid JSON in this case. 
Other suggestions for the error message: "Invalid JSON data.", "Could not load JSON", or, more generally, "Invalid project data.", "Invalid project description", "Could not understand project description".

**Comment by filippsen** [comment]
ps: Doesn't #9 relate to the "state data" mentioned here ?

File: src/components/newdapp/index.js (hunk 1 above)
Comment (Bashlund) [reply src/components/newdapp/index.js]: I was thinking the only user available option to fix the problem would be an error in copy/paste. That's why the error message suggests that. I don't have a hard stance on it though.

**Comment by Bashlund** [comment]
@filippsen you are right. I should have look more into details of @rsbondi 's approach of #9, it's probably much nicer to actually download/upload a file and then also not have the problem of copy/paste errors.
Let me do some testing and get back.

**Comment by Bashlund** [comment]
I will want to update this to use a similar approach of #9 to provide a downloadable file instead of having to copy and paste the JSON data.

**Comment by filippsen** [comment]
New code branch with the proposed changes has been added in #26

**Pull request closed without merging**